projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -207,19 +219,24 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked, OnDestro
             if (FieldsUtils.isCaseFieldOfType(casefield, ['JudicialUser'])) {
               fieldElement['component'].errors = { required: true };
             }
-          } else if (fieldElement.hasError('pattern')) {
+          }
+          if (fieldElement.hasError('pattern')) {
             this.caseEditDataService.addFormValidationError({ id, message: `%FIELDLABEL% is not valid`, label });
             fieldElement.markAsDirty();
-          } else if (fieldElement.hasError('minlength')) {
+          }
+          if (fieldElement.hasError('minlength')) {
             this.caseEditDataService.addFormValidationError({ id, message: `%FIELDLABEL% is below the minimum length`, label });
             fieldElement.markAsDirty();
-          } else if (fieldElement.hasError('maxlength')) {
+          }
+          if (fieldElement.hasError('maxlength')) {
             this.caseEditDataService.addFormValidationError({ id, message: `%FIELDLABEL% exceeds the maximum length`, label });
             fieldElement.markAsDirty();
-          } else if (fieldElement.invalid) {
-            if (casefield.isComplex()) {
+          }
+
+          if (fieldElement.invalid) {
+            if (casefield.isComplex) {
               this.generateErrorMessage(casefield.field_type.complex_fields, fieldElement, id);
-            } else if (casefield.isCollection() && casefield.field_type.collection_field_type.type === 'Complex') {
+            } else if (casefield.isCollection && casefield.field_type.collection_field_type.type === 'Complex') {
```

Comment:
You can't remove the brackets - is collection is a function, and by removing the brackets you're just testing if the member is bound to to a function, which it always will be.